graphql/main.go

```diff
@@ -0,0 +1,44 @@
+package graphql
```

Comment:
I finally checked this out to notice, but you cannot run non-main packages. Often the way you get around this is introducing an app folder that has small subdirectories each with a tiny "main.go".